src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -194,7 +194,10 @@ def nwm_output_generator(
 
     if test:
         flowveldepth.to_pickle(Path(test))
-    
+        
+        nudge = results[0][8]
+        usgs_positions_id = results[0][3][0]
```

Comment:
`results` is a list, I believe you need to loop through it to obtain all the nudge values, no?